MapboxNavigation/NavigationViewController.swift

```diff
@@ -281,6 +294,7 @@ public class NavigationViewController: NavigationPulleyViewController, RouteMapV
         super.viewWillAppear(animated)
         
         UIApplication.shared.isIdleTimerDisabled = true
+        styles.forEach { $0.apply() }
```

Comment:
I think this can also be written `styles.forEach(Style.apply)`. Whenever you find more readable.